three-layer.cabal

```diff
@@ -135,6 +135,7 @@ library
                      , time >= 1.8 && < 1.10
                      , unordered-containers
                      , warp ^>= 3.2
+                     , clock
```

Comment:
Let's also specify lower and upper bounds for this package. Also, it should be in alphabetical order in `build-depends`